### DISCUSSION TITLE
Where should 'zopen build' write logs etc?

### DISCUSSION THREAD

**Discussion by perry-ca** [zopencommunity/meta#68] [General]
I noticed building a few tools that the build step is writing log files inside of the repo of the tool being built.  For example in zoslibport we have the dirs:
- $HOME/zopen - the project dir
- $HOME/zopen/zoslibport - the dir where zoslibport is cloned into
- $HOME/zopen/zoslibport/zoslib - the dir where the actual zoslib repo is cloned

The build process is writing logs and other files (eg. config.success) into `$HOME/zoslibport/zoslib`.  This is messing up git or what ever tool we want to use to build patches or PRs.  

I think it would be better to write the log files into `$HOME/zoslibport`.  Or to create a `$HOME/zoslibport/log` and write these files into that dir.

What are people's thoughts?  I can make this change.  I just want to run it by everyone first.

**Comment by IgorTodorovskiIBM** [comment]
I like the idea of writing the logs to a dedicated log dir. We can also add `log/*` to .gitignore (and update zopen-generate to do this) so if we create more logs, they'll all be ignored.

**Reply by MikeFultonDev** [reply]
Works for me.
